samples/kubernetes/end2end/docs/06-grafana.md

```diff
@@ -9,9 +9,10 @@ Create the Grafana administrative credentials.
 ```
 kubectl --namespace monitoring create secret generic grafana-secret --from-literal=username=admin --from-literal=password=12345678
 ```
-Install the Grafana chart.
+Install the Grafana chart, in this sample we will use helm chart version 6.38.6..
```

Comment:
helm  -> Helm 
typo ->two periods at the end of the sentence.

Reply:
done